orbit/pkg/update/options.go

```diff
@@ -54,6 +54,20 @@ var (
 		},
 	}
 
+	WindowsArm64Targets = Targets{
+		constant.OrbitTUFTargetName: TargetInfo{
+			Platform:   "windows-arm64",
+			Channel:    "stable",
+			TargetFile: "orbit.exe",
+		},
+		// NOTE: Currently osquery doesn't fully support ARM64, continue to use the x86 version for now
```

Comment:
Am curious does the Windows arm64 provided in the Github releases not work?

Reply:
Hey Lucas! So I'm still a little uncertain about osquery's windows ARM support. They don't have build instructions on their website and I've tried my best to build it locally but failed. This isn't a total dealbreaker but it means that we cannot submit bug fixes upstream if we find an issue with it for the time being. I've also noticed that it doesn't return the `cpu_type` from the `system_info` table, so I'm not sure what else might behave differently. I would probably suggest someone run a full test suite against it to figure out if anything else behaves differently.

Reply:
(Because the user requesting this wanted to avoid emulation.)

Reply:
@lucasmrod One problem I've noticed with the ARM version of osquery is that it doesn't report the processor information at all. I'm also not able to debug the issue because I can't compile it locally though.